dbt/models/sale/schema.yml

```diff
@@ -1,17 +1,154 @@
-        description: '{{ doc("parameter") }}'
+        tags:
+          - load_auto
+
+        columns:
+          - name: ptax_flag_original
+            description: |
+              Whether or not this sale was flagged on Q10 of the
+              PTAX-203 form (regardless of statistical deviation)
+          - name: meta_sale_document_number
+            description: '{{ doc("shared_column_document_number") }}'
+          - name: rolling_window
+            description: |
+              Rolling window period used to calculate statistics
+              for flagging this sale
+          - name: run_id
+            description: '{{ doc("shared_column_sv_run_id") }}'
+          - name: sv_is_heuristic_outlier
+            description: '{{ doc("shared_column_sv_is_heuristic_outlier") }}'
+          - name: sv_is_ptax_outlier
+            description: '{{ doc("shared_column_sv_is_ptax_outlier") }}'
+          - name: sv_is_outlier
+            description: '{{ doc("shared_column_sv_is_outlier") }}'
+          - name: sv_outlier_type
+            description: '{{ doc("shared_column_sv_outlier_type") }}'
+          - name: version
+            description: '{{ doc("shared_column_sv_version") }}'
+
+      - name: foreclosure
+        description: '{{ doc("foreclosure") }}'
+        tags:
+          - load_manual
+
       - name: group_mean
         description: '{{ doc("group_mean") }}'
+        tags:
+          - load_auto
+
+        columns:
+          - name: group
+            description: |
+              Group string used as a unique identifier.
+
+              Typically a combination of year, township, and class
+          - name: group_size
+            description: Number of properties in the group
+          - name: run_id
+            description: '{{ doc("shared_column_sv_run_id") }}'
+          - name: mean_price
+            description: Mean price of the group, in FMV
+          - name: mean_price_per_sqft
+            description: Mean price per sqft (of building) of the group, in FMV
+
+      - name: parameter
+        description: '{{ doc("parameter") }}'
+        tags:
+          - load_auto
+
+        columns:
+          - name: condo_stat_groups
+            description: |
+              Groups used to calculate flagging statistics (std. dev.)
+              for condominium (class 299, 399) properties
+          - name: dev_bounds
+            description: |
+              Boundaries for standard deviation flagging.
+
+              Sales with prices beyond these boundaries are flagged.
+          - name: earliest_data_ingest
+            description: |
+              Date of earliest sale used in validation.
+
+              This inclusive of the rolling window period used for
+              calculating statistical groups. In other words, if the earliest
+              sale to-be-flagged is 2013-12-01 and the rolling window period
+              is 9 months, then the earliest sale *used* would be 2023-03-01
```

Comment:
[**Nitpick**]: Typo here on the year.

Reply:
Fixed in 4eb71d588cf355607577748ddc6ca63bd96aa298!